rllib/algorithms/algorithm.py

```diff
@@ -1662,7 +1663,7 @@ def restore_workers(self, workers: EnvRunnerGroup) -> None:
             )
 
     @OverrideToImplementCustomLogic
-    def training_step(self) -> ResultDict:
+    def training_step(self) -> None:
```

Comment:
`Union[ResultDict, None]`?

Reply:
You are right, but let's already go all-in on the new API stack and ignore the old APIs.